src/kdc/main.c

```diff
@@ -955,6 +955,21 @@ int main(int argc, char **argv)
 
     initialize_kdc5_error_table();
 
+    retval = krb5_get_profile(kcontext, &profile);
```

Comment:
Does this mean I should be reading it from the alternate profile like max_dgram_reply_size instead, or still use profile_get_integer() with kcontext->profile?